src/tools/ColorTool/ColorTool/SchemeParsers/IniSchemeParser.cs

```diff
@@ -124,7 +124,7 @@ public ColorScheme ParseScheme(string schemeName, bool reportErrors = false)
             if (colorTable != null)
             {
                 var consoleAttributes = new ConsoleAttributes(backgroundColor, foregroundColor, popupBackgroundColor, popupForegroundColor);
-                return new ColorScheme(colorTable, consoleAttributes);
+                return new ColorScheme(schemeName.Substring(0, schemeName.Length - FileExtension.Length), colorTable, consoleAttributes);
```

Comment:
Looking at this repeated pattern, I'm searching for a way to re-use this across the parsers since it's the same logic every time...

Reply:
Easiest thing I could come up with was a common base class for the scheme parsers that handles this.  Just pushed that to the PR.  What do you think?

Reply:
Yes, I like it. Thank you.